lib/pages/new_group/new_group.dart

```diff
@@ -112,6 +113,11 @@ class NewGroupController extends State<NewGroup> {
     final client = Matrix.of(context).client;
 
     try {
+      if (nameController.text.isEmpty) {
+        setState(() => error = L10n.of(context).pleaseEnterName);
```

Comment:
There is already a translation which might fit here:

```suggestion
        setState(() => error = L10n.of(context).pleaseFillOut);
```

Reply:
Thanks, I have updated this!